columnflow/tasks/ml.py

```diff
@@ -462,7 +462,7 @@ def requires(self):
 
     @MergeReducedEventsUser.maybe_dummy
     def output(self):
-        return self.target(f"mlcols_{self.branch}.pickle")
+        return {"mlcols": self.target(f"mlcols_{self.branch}.pickle")}
```

Comment:
For consistency with "events", "mlevents", and "columns", we should maybe call this "mlcolumns" (same then in the other places below).

Reply:
Yeah, I agree. Should we then also change the file name to "mlcolumns_{branch}.pickle"?